scripts/windows/choco/signalfx-agent/signalfx-agent.nuspec

```diff
@@ -9,21 +9,32 @@ The SignalFx Smart Agent is a metric-based agent written in Go that is used to m
 #### Package Parameters
 The following package parameters can be set:
 
- * `/access_token` - The token used to send metric data to SignalFx. Required if C:\ProgramData\SignalFxAgent\token does not exist.
- * `/ingest_url` - Base URL of the SignalFx ingest server. Defaults to 'https://ingest.signalfx.com'.
- * `/api_url` - Base URL of the SignalFx API server. Defaults to 'https://api.signalfx.com'.
+ * `/access_token` - The token used to send metric data to SignalFx. If not specified, the value in the `\ProgramData\SignalFxAgent\token` file (if it exists) will be used. See the **note** below for details regarding the token and starting the SignalFx Smart Agent service after installation.

```

Comment:
See my previous comments for this section. I think it's identical to one at the beginning of the file.